nbs/models.ipynb

```diff
@@ -57,6 +57,7 @@
     "import pandas as pd\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #56.            out = _historic_average(y=y, h=h, fitted=True)
</pre><p>Instead of <code>fitted=True</code>, I think we can use:</p><p> </p><pre class="ql-syntax" spellcheck="false">fitted = fitted or (level is not None)
out = _historic_average(y=y, h=h, fitted=fitted)
</pre><p>   </p><p>Then, if the fitted values are necessary for the calculation of the prediction intervals, <code>_historic_average</code> will return them even if <code>fitted=False</code>. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/statsforecast/pull/201/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='89'/>